schemas/v2/schema.json

```diff
@@ -203,7 +203,7 @@
       "additionalProperties": {
         "$ref": "#/definitions/response"
       },
-      "description": "One or more JSON representations for parameters"
+      "description": "One or more JSON representations for responses"
```

Comment:
ok with fixing the typo. Could you please update the bindata.go generated file?

Reply:
Sure, but I see a bunch of unrelated go-bindata changes as a result. Are you using jteeuwen/go-bindata, or a different fork/version?

Reply:
To be honest, this one hasn't been changed in a while... I would expect a big change here.
To be consistent with what we are using for the go-swagger repo, best is to use kevin burke's fork: https://github.com/kevinburke/go-bindata

Reply:
Added the refs to kevinburke's and regenerated.